components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2527,8 +2527,20 @@ def _table_query(request, fileid, conn=None, **kwargs):
         except Exception:
             return dict(error='Error executing query: %s' % query)
 
+    colDescriptions = []
+    for col in cols:
+        desc = col.description
+        # description might be json data
+        try:
+            if len(desc) > 0:
+                desc = json.loads(desc)
+        except:
+            pass
+        colDescriptions.append(desc)
+
     return dict(data=dict(
         columns=[col.name for col in cols],
+        descriptions=colDescriptions,
```

Comment:
Actually, I don't think you're using any of the column descriptions (and json parsing of them), are you? If not then this could all be removed.